src/rmodels.c

```diff
@@ -4491,6 +4497,10 @@ static ModelAnimation *LoadModelAnimationsIQM(const char *fileName, unsigned int
     //fread(framedata, iqmHeader->num_frames*iqmHeader->num_framechannels*sizeof(unsigned short), 1, iqmFile);
     memcpy(framedata, fileDataPtr + iqmHeader->ofs_frames, iqmHeader->num_frames*iqmHeader->num_framechannels*sizeof(unsigned short));
 
+    // joints
+    IQMJoint* joints = RL_MALLOC(iqmHeader->num_joints * sizeof(IQMJoint));
```

Comment:
Please, could you review raylib coding style, `*` pointer should be placed in the variable and multiplications have no spaces.